pkg/vulnsrc/vulnsrc.go

```diff
@@ -47,15 +49,31 @@ func Update(names []string) error {
 
 	// Clone vuln-list repository
 	dir := filepath.Join(utils.CacheDir(), "vuln-list")
-	updatedFiles, err := git.CloneOrPull(repoURL, dir)
-	if err != nil {
-		return xerrors.Errorf("error in vulnsrc clone or pull: %w", err)
-	}
-	log.Logger.Debugf("total updated files: %d", len(updatedFiles))
+	//updatedFiles, err := git.CloneOrPull(repoURL, dir)
+	//if err != nil {
+	//	return xerrors.Errorf("error in vulnsrc clone or pull: %w", err)
+	//}
+	//log.Logger.Debugf("total updated files: %d", len(updatedFiles))
+	//
+	//// Only last_updated.json
+	//if len(updatedFiles) <= 1 {
+	//	return nil
+	//}
```

Comment:
Remove if not needed.

Reply:
Sorry. this PR is still WIP. These codes are necessary, but they are commented out for debug.